DOCKER.md

```diff
@@ -0,0 +1,181 @@
+
+# Quick reference
+
+<p align="center">
+  Maintained by <a href="https://www.localstack.cloud/">LocalStack</a>,  with contributions from our  <a href="https://www.localstack.cloud/" target="_blank">community</a> </p>
+<p align="center">
+  <a href="https://docs.localstack.cloud" target="_blank">📖 Docs</a> •
+  <a href="https://app.localstack.cloud" target="_blank">💻 Pro version</a> •
+  <a href="https://docs.localstack.cloud/references/coverage/" target="_blank">☑️ LocalStack coverage</a>
+</p>
+
+### Where to get help 
+
+Get in touch with the LocalStack Team to report 🐞 [issues](https://github.com/localstack/localstack/issues/new/choose),upvote 👍 [feature requests](https://github.com/localstack/localstack/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc+),🙋🏽 ask [support questions](https://docs.localstack.cloud/getting-started/help-and-support/),or 🗣️ discuss local cloud development:
+
+- [LocalStack Slack Community](https://localstack.cloud/contact/)
+- [LocalStack Discussion Page](https://discuss.localstack.cloud/)
+- [LocalStack GitHub Issue tracker](https://github.com/localstack/localstack/issues)
+- [Getting Started - FAQ](https://docs.localstack.cloud/getting-started/faq/)
+
+##Base Image Tags
```

Comment:
```suggestion
## Base Image Tags
```

Reply:
Thank you for the review, sorry about that I'll commit these fixes along with other formatting issues I found.